validphys2/src/validphys/filters.py

```diff
@@ -45,6 +45,41 @@
 }
 
 
+def _get_kinlabel_process_type(process_type):
+    """Get KIN_LABEL from the dictionary above according
+    to the process type
+    This requires some extra digestion for DIS
+    """
+    if process_type[:3] == "DIS":
+        return KIN_LABEL["DIS"]
+    return KIN_LABEL[process_type]
+
+
+# TODO: in the new commondata instead of having this, let's always use the same
+# variables
+def _variable_understanding(variables_raw, process_vars):
+    """Given a set of variable, check whether it might be a variation of existing
+    variables for a process type"""
+    variables = [i for i in variables_raw]
+
+    def substitute(pr_v, cd_x):
+        if pr_v in process_vars and cd_x in variables:
+            variables[variables.index(cd_x)] = pr_v
+
+    substitute("eta", "y")
+    substitute("eta", "eta")
+    substitute("etay", "eta")
+    substitute("etay", "y")
+    substitute("p_T2", "pT_sqr")
+    substitute("sqrts", "sqrt_s")
+    substitute("sqrt(s)", "sqrts")
+    substitute("sqrt(s)", "sqrt_s")
+    substitute("yQQ", "y_ttBar")
+    substitute("yQ", "y_t")
```

Comment:
I will do a search & replace of all the variables in `cuts/filters.yaml` and this module in a later PR, since that will break runcards using extra filter rules.

For now I will remove the ones that I know are not necessary anymore.